lib/eco/treemanager.py

```diff
@@ -127,6 +128,8 @@ def right(self):
 
     def jump_left(self):
         self.node = self.find_previous_visible(self.node)
+        if self.node.symbol.name == "\r":
+            self.line -= 1
```

Comment:
That's a bit strange. This has been already merged into master but it doesn't show up in this comparison.